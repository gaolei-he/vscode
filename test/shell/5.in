81
/bbb/dad
/d
/bdd/a
/dbb/d/cb
/cc/c/d
/bbb/c/c
/ccd
/d
/cb/d/c
/c/acc/a
/ac/a/bb
/a/d/d
/d
/a
/a/cbd
/dd
/abd
/cc/cda/b
/ccb/b
/d/cad
/ccc
/cd/a
/dd
/d
/bc/c
/d/dc/db
/cd
/bdc/dcb/ab
/bdc/aba/abd
/a/a/a
/abb/d
/cdc/dbd
/bdc
/bc/cb
/b/cc/ab
/d
/aaa
/ca/d/abb
/b/a
/ac
/dab/ba
/a
/abc/b
/d/caa/cbc
/cab
/b/dcd
/ca/cba/ba
/b/bb/bb
/dc
/b/ba/da
/b
/cc
/a
/cba
/bcc
/bbd/c/b
/d/bda
/cca/a/d
/b/c
/cbc/c
/bab
/d/ccd
/bbc
/cca/b
/ba
/ac
/ccc/bcd/a
/ac/b
/c/dd
/c
/b/c
/bd/c/ba
/bcb
/bbc
/c/aca
/bac/c
/bdc/dd/dab
/da/ccc
/cbd/dc/ad
/bbd/acc/cb
/dd/aa/daa
52
ls
cd .
ls
ls
ls
ls
pwd
ls
cd ..
ls
pwd
cd .
pwd
pwd
pwd
ls
ls
cd ..
cd ..
cd .
ls
ls
cd ..
pwd
pwd
cd ..
ls
cd .
ls
ls
cd cbc
cd .
ls
ls
pwd
pwd
ls
ls
ls
ls
pwd
cd ..
pwd
pwd
ls
pwd
pwd
pwd
pwd
cd .
cd .
ls
