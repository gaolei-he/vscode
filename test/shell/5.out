a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
/
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
/
/
/
/
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
/
/
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
a
aaa
abb
abc
abd
ac
b
ba
bab
bac
bbb
bbc
bbd
bc
bcb
bcc
bd
bdc
bdd
c
ca
cab
cb
cba
cbc
cbd
cc
cca
ccb
ccc
ccd
cd
cdc
d
da
dab
dbb
dc
dd
c
c
/cbc
/cbc
c
c
c
c
/cbc






